### PR TITLE
Fix kebab-case filename & Add Boku no hero

### PR DESCRIPTION
**Pull request by mohfirdaus**
[x] 🔍 I have searched the datasource directory and confirmed I am not adding a duplicate title.
[x] 🌻 I have added a anime year under 2016. 
[x] 🔗 I have added the myanimelist page under https://myanimelist.net/anime/31964/Boku_no_Hero_Academia.
[x]  3️⃣ I have at least three emojis listed under animeEmoji.
[ ] 8️⃣ I have a maximum of eight emojis listed under animeEmoji.
[x] 🥙 I have written the file using kebab-case.
[x] 👍 My pull request has a descriptive title.

### CODE REVIEW

**Comment by mohfirdaus** [comment]
I have problem with kebab-case. But my file was used kebab-case filename. Can you tell me what wrong with my RP? Thanks :)

File: src/datasource/Boku no Hero.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (2pai) [inline src/datasource/Boku no Hero.json:1]: I think this file caused an error

**Review by 2pai — CHANGES_REQUESTED** [review]

File: src/datasource/Boku no Hero.json (hunk 1 above)
Comment (mohfirdaus) [reply src/datasource/Boku no Hero.json]: Ok thanks, I will try to solve this problem

Comment (mohfirdaus) [reply src/datasource/Boku no Hero.json]: I was deleted the file from this PR. But the problem wasn't solved. And I got an error below

"Detective-Conan is not in the right pattern. Please use kebab-case.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! animoji@0.1.0 validate:file-name: `node src/datasource/validate.js file-name`
npm ERR! Exit status 1"

Comment (2pai) [reply src/datasource/Boku no Hero.json]: i think ur codebase is not the same as in the upstream, can you rename the Detective-Conan to detective-conan?

Comment (mohfirdaus) [reply src/datasource/Boku no Hero.json]: I give up :(. I need learn more about Git. But thanks :)

Comment (2pai) [reply src/datasource/Boku no Hero.json]: that's okay, just rename the file Detective-Conan to detective-conan.

Comment (mohfirdaus) [reply src/datasource/Boku no Hero.json]: I was rename all the namefile using kebab-case but the problem is not resolved

Comment (2pai) [reply src/datasource/Boku no Hero.json]: have u rename detective-conan and commit the changes?

Comment (pr0ph0z) [reply src/datasource/Boku no Hero.json]: Hi there, I think there's a little mistake I made. I haven't renamed the `Detective-Conan` to `detective-conan`. You can try to rename the file and try to push a commit again.

Comment (mohfirdaus) [reply src/datasource/Boku no Hero.json]: It's solved :). I was rename to detective-conan but the problem is not resloved. But now I rename to the japanese name (meitantei-konan) and solved.

Comment (2pai) [reply src/datasource/Boku no Hero.json]: hi, can you use detective-conan.json instead of meitantei-konan.json ?

Comment (pr0ph0z) [reply src/datasource/Boku no Hero.json]: Nice solving! Thanks for contributing.

**Comment by 2pai** [comment]
Congratulations @mohfirdaus !
Thanks for the contribution :D 👍

File: src/datasource/Boku no Hero.json (hunk 1 above)
Comment (mohfirdaus) [reply src/datasource/Boku no Hero.json]: Done

**Review by 2pai — APPROVED** [review]
LGTM

**Comment by 2pai** [comment]
"the commits are kind of messy but we don't mind too much about it. For further learning, you can try about git squash to 'combine' the commits in your local git"

**Comment by mohfirdaus** [comment]
> "the commits are kind of messy but we don't mind too much about it. For further learning, you can try about git squash to 'combine' the commits in your local git"

Thanks For The Insight :)

**Pull request merged**